data/src/main/kotlin/remote/RemoteRepositoryImpl.kt

```diff
@@ -0,0 +1,47 @@
+package remote
+
+import com.bbuddies.gotogether.common_domain.model.Message
+import com.bbuddies.gotogether.common_domain.model.Reply
+import com.bbuddies.gotogether.common_domain.model.User
+import com.bbuddies.gotogether.common_domain.repository.RemoteRepository
+import remote.api.RetrofitInstance
+
+class RemoteRepositoryImpl() : RemoteRepository {
+    private val webService = RetrofitInstance.madafakerWebService
+
+    override suspend fun getCurrentUser(): User {
+        return webService.getCurrentUser()
+    }
```

Comment:
Just notice: expression func notation is a bit more casual for Kotlin ;)
`override suspend fun getCurrentUser(): User =
        webService.getCurrentUser()
    `

Reply:
No changes needed

Reply:
No  changes needed